src/bindings/html/langs.js

```diff
@@ -18,7 +18,7 @@ export function getAdditionalLanguages() {
 export function onlanguagechage(
   appVersion, defaultLang, availableLangs, requestedLangs) {
 
-  return this.languages = Promise.all([
+  return Promise.all([
```

Comment:
So in general, I don't think we should be overloading `prevLangs` for the pretranslation optimization.  Could we instead simply check in `translateDocument` if there's already `lang` set on `documentElement` and if it's the same as `langs[0]` skip directly to `setDOMLocalized`?